res/style.css

```diff
@@ -149,10 +152,6 @@ body, #root, .profileViewer {
   flex-flow: column nowrap;
   align-items: center;
 }
-.treeViewRowColumn {
-  overflow: hidden;
-  text-overflow: ellipsis;
-}
```

Comment:
This makes the box-shadow visible again.

Reply:
The text-overflow is also what was triggering the issue, but we don't need it in the main column as we don't have a width